Assets/PatchKit Patcher/Scripts/UI/Languages/PatcherLanguages.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Newtonsoft.Json;
+using PatchKit.Unity.Patcher.Debug;
+using PatchKit.Unity.Utilities;
+using UnityEngine;
+
+namespace PatchKit.Unity.UI.Languages
+{
+    public class PatcherLanguages : MonoBehaviour
+    {
+        public static PatcherLanguages Instance;
+
+        public static Dictionary<String, String> Fields;
+
+        [SerializeField] 
+        string defaultLanguage = "en";
```

Comment:
`[SerializeField]` should be scraped, `static` should be added and the name captialized.

Reply:
done